vllm/entrypoints/openai/api_server.py

```diff
@@ -72,10 +73,17 @@ async def get_gen_prompt(request) -> str:
             "fastchat is not installed. Please install fastchat to use "
             "the chat completion and conversation APIs: `$ pip install fschat`"
         )
+    if fastchat.__version__ < "0.2.23":
```

Comment:
Can you compare the version using a more robust library? Example: https://stackoverflow.com/a/11887885